src/Usuarios.php

```diff
@@ -85,7 +85,46 @@ public static function updateUsuario(
         \Psr\Http\Message\ResponseInterface $respuesta,
         $argumentos
     ) {
-        // TODO: Actualizar un usuario por su identificador usando los datos enviados en el formulario
+        /* Configuramos el tipo MIME correcto para una salida JSON */
+        $respuesta = $respuesta->withHeader('Content-Type', 'application/json');
+        $body = $respuesta->getBody();
+        try {
+            /* Actualizamos el usuario seleccionado por su identificador, usando los datos enviados en el formulario */
+            $conexion = \miPDO\Conexion::obtenerPDO();
```

Comment:
__[phpmd]__
* Avoid using static access to class '\miPDO\Conexion' in method 'updateUsuario'.